### PR TITLE
add in some more underling textures

### PR DESCRIPTION
**Pull request by InfectedGriffon**
Completes the roster of underling grist textures.

All art by Riotmode

### CODE REVIEW

**Comment by kirderf1** [comment]
Is the PR ready, or is there anything that's left before it is?

**Comment by InfectedGriffon** [comment]
oh yeah sorry that's all of them I think

File: src/main/java/com/mraof/minestuck/client/renderer/entity/UnderlingRenderer.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import software.bernie.geckolib3.renderers.geo.IGeoRenderer;
 
 import java.io.IOException;
+import java.util.Arrays;
```
Comment (Dweblenod) [inline src/main/java/com/mraof/minestuck/client/renderer/entity/UnderlingRenderer.java:28]: Remove unused import

**Review by Dweblenod — COMMENTED** [review]
Once the unused import is removed, this will be ready to add in my mind!

**Pull request merged**